test/test-rep.js

```diff
@@ -0,0 +1,36 @@
+var StringBuilder = require('../src/string-builder');
+var assert = require('chai').assert;
+var sinon = require('sinon');
+var jsonfile = require('jsonfile');
+
+var file = './fixtures/data.json';
+var fixtures = jsonfile.readFileSync(file);
+
+describe('StringBuilder #rep', function() {
+    it('Should add twice the paramaters', function() {
+        var sb = new StringBuilder();
+
+        sb.rep('hello','world', 2);
+
+        assert.deepEqual(sb.string(), fixtures.twiceElements.join(''));
+    });
+
+    it('Should add parameters n times and using chaining patters', function() {
+        var sb = new StringBuilder();
+
+        sb.cat('Can I go,')
+            .rep('please ', 2)
+            .rep('?',3)
+
+        assert.deepEqual(sb.string(), 'Can I go,please please ???');
+    });
+
+    it('Should throw error for howManyTimes not a number', function() {
+        var sb = new StringBuilder();
+        function callback() {
```

Comment:
We are devoted lovers of **arrow functions** because they shorten our code and make it look nicer